README.md

```diff
@@ -48,6 +48,12 @@ Currently the most reliable way of installing Gatekeeper is to build and install
    * make sure your kubectl context is set to the desired installation cluster
    * run `make deploy`
 
+If you want to skip the long way and just quickly deploy Gatekeeper in your cluster with a prebuilt image, then you can run the following command:
```

Comment:
- "skip the long way and just quickly"

I think the important difference here is HEAD vs release. Relative effort should be apparent from instructions.